lib/node_modules/@stdlib/blas/ext/base/sasumpw/src/addon.c

```diff
@@ -35,14 +35,27 @@
 static napi_value addon( napi_env env, napi_callback_info info ) {
 	STDLIB_NAPI_ARGV( env, info, argv, argc, 3 );
 	STDLIB_NAPI_ARGV_INT64( env, N, argv, 0 );
-	STDLIB_NAPI_ARGV_INT64( env, stride, argv, 2 );
-	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, stride, argv, 1 );
-
-	napi_value v;
-	napi_status status = napi_create_double( env, stdlib_strided_sasumpw( N, X, stride ), &v );
-	assert( status == napi_ok );
+	STDLIB_NAPI_ARGV_INT64( env, strideX, argv, 2 );
+	STDLIB_NAPI_ARGV_STRIDED_FLOAT32ARRAY( env, X, N, strideX, argv, 1 );
+	STDLIB_NAPI_CREATE_DOUBLE( env, API_SUFFIX(stdlib_strided_sasumpw)( N, X, strideX ), v );
```

Comment:
```suggestion
	STDLIB_NAPI_CREATE_DOUBLE( env, (double)API_SUFFIX(stdlib_strided_sasumpw)( N, X, strideX ), v );
```

Explicit cast of return value.